articles/migrations/guides/liveid-api-deprecation.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Migration to LinkedIn API V2
+description: This article covers the LinkedIn API deprecation and how to update your Auth0 LinkedIn Connection.
+toc: true
+contentType:
+  - how-to
+useCase:
+  - add-login
+  - migrate
+---
+
+# Microsoft Account Migration to Azure AD (personal accounts) + Microsoft Graph
+
+In October 2017, Microsoft announced the [deprecation of the Live Connect API and Live SDK](https://developer.microsoft.com/en-us/office/blogs/outlook-rest-api-v1-0-office-365-discovery-and-live-connect-api-deprecation). This is a Microsoft deprecation that will affects Auth0 users using the Microsoft social connection, and requires changes in the Auth0 configuration and potentially the application code. 
+
+The change implies switching:
+
+- from the Live Connect API to the Azure Active Directory API to authenticate
+- from the Live SDK to Microsoft Graph to be able to get to other resources including user profiles, contacts, files, etc
```

Comment:
to get to other -> to get other (both are valid, though, one just sounds better to me)